Cargo.lock

```diff
@@ -3335,8 +3255,8 @@ dependencies = [
  "aws-config",
  "aws-lc-rs",
  "aws-sdk-kms",
- "aws-smithy-experimental",
- "aws-smithy-http",
+ "aws-smithy-http 0.60.11",
```

Comment:
I see that we have multiple `aws-smithy-http`'s here but no corresponding `skip` in `deny.toml`. Any idea why?

Reply:
No, that's really strange, I'm not sure how that wasn't flagged by `cargo-deny`! The `0.60.x` version comes from `tough-kms`'s dev-dependencies, where we specifically use that version. I'll bump it to `0.62`.

Reply:
I've noticed sometimes the `aws-sdk-rust` crates will get ahead of themselves and pull multiple `aws-smithy-http`s. You might be able to shake it by pulling newer (but not the newest) SDK crates.